qa/tasks/vstart_runner.py

```diff
@@ -769,23 +768,10 @@ def _set_fuse_daemon_pid(self, check_status):
 # XXX: this class has nothing to do with the Ceph daemon (ceph-mgr) of
 # the same name.
 class LocalCephManager(CephManager):
-    def __init__(self):
-        # Deliberately skip parent init, only inheriting from it to get
-        # util methods like osd_dump that sit on top of raw_cluster_cmd
-        self.controller = LocalRemote()
-
-        # A minority of CephManager fns actually bother locking for when
-        # certain teuthology tests want to run tasks in parallel
-        self.lock = threading.RLock()
-
+    def __init__(self, controller, ctx):
+        super().__init__(controller, ctx)
```

Comment:
I'm not sure this won't break a lot of tests. @rishabh-d-dave do you have any thoughts?

Reply:
I do intend to test this PR myself but I haven't been able to get a healthy vstart cluster. I'll get back to this as soon as I can get one.

Reply:
Yeah, sure, will fix it.